client/dashboard/sites/index.tsx

```diff
@@ -214,7 +227,19 @@ export default function Sites() {
 						fields={ fields }
 						actions={ actions }
 						view={ view }
-						onChangeView={ setView }
+						onChangeView={ ( view ) => {
+							if ( view.type === 'list' ) {
+								return;
+							}
+							const _defaultView = { ...defaultView, ...( DEFAULT_LAYOUTS[ view.type ] ?? {} ) };
```

Comment:
```suggestion
							const _defaultView = { ...defaultView, ...DEFAULT_LAYOUTS[ view.type ] };
```

works even with undefined values.

Reply:
Yes, I think I was trying to figure out what made TS unhappy :)